features/parametervalidatie.feature

```diff
@@ -108,7 +109,7 @@ Functionaliteit: parameters worden gevalideerd
     Dan levert dit een foutmelding
     En heeft de foutmelding betrekking op parameter verblijfplaats__gemeentevaninschrijving
 
-  Scenario: Bij zoeken op voorvoegsel moet een waarde worden gebruikt die voorkomt in de betreffende GBA tabel of moet leeg zijn
+  Scenario: Bij zoeken op voorvoegsel moet een waarde worden gebruikt die voorkomt in de betreffende GBA tabel, waardetabel of moet leeg zijn
```

Comment:
wat betekent hier de toevoeging "waardetabel"?

bij nader inzien vraag ik me ook af hoe zinvol het is om:
1. voorvoegsels te valideren op voorkomen in de tabel
2. een tabel in de BRP tabellen API op te nemen voor voorvoegsels

Ad 1. je zoekt hier niet echt op een tabelwaarde, want het zoeken is case insensitive en je geeft hier de omschrijving terug (er is geen code). Bijvoorbeeld zoeken met naam__voorvoegsel=van der moet ook een persoon kunnen vinden met naam.voorvoegsel=Van Der. Als iemand bijvoorbeeld een applicatie maakt waarin je een persoon kan opzoeken met naam en voorvoegsel (plus geboortedatum), moet de gebruiker een voorvoegsel in kunnen typen. Dan is het onhandig om dan ook een specifiek voorvoegsel uit de tabel te selecteren o.i.d. Anders is dat bijvoorbeeld bij zoeken op gemeente(code), want daar is er wel een code en moet je anders exact de juiste schrijfwijze van de gemeente gebruiken die niet altijd bekend is bij een gebruiker (bijvoorbeeld 's-Gravenhage en niet 's Gravenhage).

Ad 2. de voorvoegseltabel is de enige tabel met alleen waarden/omschrijvingen, geen codes. Deze tabel ter beschikking stellen heeft erg weinig zin denk ik.

Reply:
besproken in overleg. besloten dat de voorvoegsel queryparameter niet gevalideerd wordt op voorkomen in landelijke tabel voorvoegseltabel.

Reply:
Deze validatie is verwijderd uit de feature parametervalidatie